packages/framework/src/Model/Mail/MailerSettingProvider.php

```diff
@@ -24,28 +19,17 @@ class MailerSettingProvider
     protected bool $deliveryDisabled;
 
     /**
-     * @param string $mailerWhitelist
      * @param string $mailerMasterEmailAddress
      * @param string $mailerDsn
      */
     public function __construct(
-        string $mailerWhitelist,
```

Comment:
Because of https://github.com/shopsys/shopsys/pull/2592#discussion_r1166260182 this variable stay intact, but other one `$mailerMasterEmailAddress` was deprecated in backward compatible way.